includes/class-sensei-teacher.php

```diff
@@ -291,11 +291,13 @@ public function teacher_meta_box_content( $post ) {
 	 * @return array $users user id array
 	 */
 	public function get_teachers_and_authors() {
+		global $wp_version;
 
+		$authors_field     = version_compare( $wp_version, '5.9.0', '>=' ) ? 'capability' : 'who';
```

Comment:
Note: Looks like the `version_compare` is not returning true when the `$wp_version` is `5.9-beta1`. I had to set `$wp_version = '5.9.0'` to make it work.